timer.go

```diff
@@ -36,7 +36,7 @@ type labeledTimer struct {
 }
 
 func (lt *labeledTimer) WithValues(labels ...string) Timer {
-	return &timer{m: lt.m.WithLabelValues(labels...)}
+	return &timer{m: lt.m.WithLabelValues(labels...).(prometheus.Histogram)}
```

Comment:
What is going on here? Was there a problem?

Reply:
yes, due to the api change, it cannot compile against prom master now:
https://github.com/docker/go-metrics/issues/12

Reply:
Please send a separate PR. We may need to change the field type instead of adding a type assertion.